packages/blade/src/components/Accordion/Accordion.tsx

```diff
@@ -0,0 +1,57 @@
+import type { ReactElement } from 'react';
+import { cloneElement, Children } from 'react';
+import { BaseBox } from '~components/Box/BaseBox';
+import type { StyledPropsBlade } from '~components/Box/styledProps';
+import type { TestID } from '~src/_helpers/types';
+
+type AccordionProps = {
+  /**
+   * Makes the passed item index expanded by default (uncontrolled)
+   */
+  defaultExpandedIndex?: number;
+
+  /**
+   * Expands the passed index (controlled)
+   */
+  expandedIndex?: number;
+
+  /**
+   * Callback for change in any item's expanded state
+   */
+  onChange?: ({ expandedIndex }: { expandedIndex: number | undefined }) => void;
```

Comment:
When can it be undefined?

Reply:
When expandedIndex becomes undefined? should we make it so that if expandedIndex is -1 then means no items are expanded?

Reply:
On point. I realized later making this work for controlled behavior. Updated the logic to `-1` (along with API doc).